### ISSUE TITLE
Issues installing locally

### ISSUE THREAD

**Issue by simplenotezy** [TeamWertarbyte/material-ui-snackbar-provider#29]
I was trying to clone down the repository to submit a PR for support for react 17, however, when running npm install in this projects directory I get:

```
[mf:...terial-ui-snackbar-provider]$ npm install                                                                                                                                                 (master✱)

> material-ui-snackbar-provider@1.4.0 prepare /Users/mf/Projects/material-ui-snackbar-provider
> babel src -d lib && rm lib/*.test.js && (bash -c 'shopt -s globstar && cd src && cp --parents -vt ../lib/ **/*.d.ts')

Browserslist: caniuse-lite is outdated. Please run the following command: `npx browserslist --update-db`
Successfully compiled 9 files with Babel.
bash: line 0: shopt: globstar: invalid shell option name
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! material-ui-snackbar-provider@1.4.0 prepare: `babel src -d lib && rm lib/*.test.js && (bash -c 'shopt -s globstar && cd src && cp --parents -vt ../lib/ **/*.d.ts')`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the material-ui-snackbar-provider@1.4.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mf/.npm/_logs/2021-07-01T07_29_27_546Z-debug.log
```

Not sure how to proceed from here

**Comment by leMaik** [comment]
tbh I'm not even sure what that command is supposed to do :sweat_smile: 
I'll see if I can get rid of it

**Comment by leMaik** [comment]
Fixed in v1.4.1
